packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -636,6 +662,28 @@ abstract class Page<T> extends RouteSettings {
   ///    Flutter.
   final String? restorationId;
 
+  /// Called after a pop on the associated route was handled.
```

Comment:
Nit: Is there anything that could be deduplicated with a macro in these docs?  Here and for `canPop`.